.flake8

```diff
@@ -0,0 +1,3 @@
+[flake8]
```

Comment:
duplicate file? I saw this file somewhere.

Reply:
Yes -- merged the other PR, updated this branch, and there is no more change to .flake8.